.github/workflows/caching-build.yml

```diff
@@ -129,9 +170,55 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
         with:
-          shared-key: ${{ env.SCHED_EXT_KERNEL_COMMIT }}
           workspaces: rust
+          key: ${{ matrix.component }}
+          prefix-key: "1"
+      - run: cargo build  --manifest-path rust/${{ matrix.component }}/Cargo.toml
+      - run: cargo test --manifest-path rust/${{ matrix.component }}/Cargo.toml --no-run
+      - run: vng -v --memory 10G --cpu 8 --force-9p -r linux/arch/x86/boot/bzImage  --net user -- cargo test --manifest-path rust/${{ matrix.component }}/Cargo.toml
```

Comment:
I think we should drop `--force-9p` also from here.

Reply:
lol ofc I missed the instance of that in the file I was editing the most, thx

Reply:
ahh forgot rg skips hidden directories and `.github` is one, thx.